app/controllers/reviews_controller.rb

```diff
@@ -15,7 +15,7 @@ def create
       redirect_to dinosaur_path(@dinosaur)
     else
       flash[:errors] = @review.errors.full_messages.join(". ")
-      render :'dinosaurs/show'
+      redirect_to dinosaur_path(@dinosaur)
```

Comment:
Use `render 'dinosaurs/show'` so that the form includes the values previously entered.